### DISCUSSION TITLE
regarding use of neper

### DISCUSSION THREAD

**Discussion by anujyadav0693** [neperfepx/neper#874] [Q&A] [answered]
respected sir,
I am using Neper to make a bimodal microstructure.
neper -T -n from_morpho -morpho "diameq:0.1*lognormal(0.0005,0.00005)+lognormal(0.00005,0.000005),1-sphericity:lognormal(0.145,0.03)" -group "diameq>0.00009?1:2" -morphooptiini "weight:radeq,coo:packing" -dim 2 -domain "square(0.03,0.01)" -o void -format "geo,ori,tess" -statcell diameq -oridescriptor "euler-bunge" -ori "fiber(0,0,1,0,0,1)" -statgroup area -statface id,area

Sir, I am using this code, and I am still getting 3.75% of the area following the small grain distribution, and the remaining area is given to large grain. This also takes too much time to run. I am running in HPC in 40 threads. Job even not complete within 3 days and failed due to time out.
one of your discussion i found that by multiplying 0.8*radeq reduce the time but it also take long time and job terminated due to less memory. 

Sir, I want to make a model with a high porosity maximum of 20% per cent, but I cannot model it. Please help me do this. Is there any other way to increase the porosity percentage?  I know the rmcell command is working in the latest version. Please guide how I can get a model with high porosity. To reduce time as well.

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
